src/tokens.md

```diff
@@ -88,6 +88,20 @@ evaluated (primarily) at compile time.
 
 #### Suffixes
 
+Any kind of literal (string, integer, etc) with any suffix is valid as a token,
```

Comment:
The suffix must be an IDENT, right?

e.g. `blackhole!("string"4);` is not OK even tho `4` is a suffix of `"string"`

Reply:
>The suffix must be an IDENT, right?

Yes, added a commit with the definition.

Reply:
`4` is not a suffix, `"string"4` is lexed into two literal tokens.